postgresql_extensions/src/repository/portal_corp/matcher.rs

```diff
@@ -0,0 +1,89 @@
+use postgresql_archive::Result;
+use semver::Version;
+use std::collections::HashMap;
+use url::Url;
+
+/// Matcher for Portal Corp binaries from <https://github.com/portalcorp>
```

Comment:
I might suggest listing the repo (https://github.com/portalcorp/pgvector_compiled/) rather than the org link as we might be adding more repos so it could get confusing.

Reply:
I used the GitHub organization in case you end up adding more extensions to the organization they can easily be added.  Feel free to submit a PR with any changes you feel would provide clarity.  I may also considering refactoring the extension matchers and repository extraction logic to simplify/encourage adding new extensions.